src/resolve/resolved-dns-server.c

```diff
@@ -719,9 +719,9 @@ DnsServer *manager_set_dns_server(Manager *m, DnsServer *s) {
                 return s;
 
         if (s)
-                log_info("Switching to %s DNS server %s.",
-                         dns_server_type_to_string(s->type),
-                         dns_server_string(s));
+                log_debug("Switching to %s DNS server %s.",
+                          dns_server_type_to_string(s->type),
+                          dns_server_string(s));
```

Comment:
I think it's OK to downgrade the log for now, but that's just masking a real issue. In my logs I have resolved switching between 8.8.8.8 and 8.8.4.4 thousands of times, back and forth. I didn't ever look in the reason why, but I doubt it should be doing that.